measurements/filestore.py

```diff
@@ -1,12 +0,0 @@
-import os
-import errno
-
-from six.moves.urllib.parse import urlparse, urljoin
-
-from sqlalchemy import exists
-
-def get_download_url(app, bucket_date, filename):
-    url = "https://s3.amazonaws.com/"
-    # strip the leading s3://
-    url += app.config['REPORTS_DIR'][5:]
-    return urljoin(url, bucket_date, filename)
```

Comment:
Okay, this is not necessary anymore. I am going to `git grep REPORTS_DIR` to understand whether it makes sense to keep it, or not.

Reply:
@hellais:

```
$ git branch
* feature/download_url
  master
$ git grep -n REPORTS_DIR
Readme.md:88:`REPORTS_DIRECTORY` in the configuration file.
config/development.env:2:REPORTS_DIR=s3://ooni-public/sanitised
config/example-secrets.env:11:REPORTS_DIR=s3://ooni-public/sanitised
config/production.env:2:REPORTS_DIR=s3://ooni-public/sanitised
config/staging.env:2:REPORTS_DIR=s3://ooni-public/sanitised
config/testing.env:2:REPORTS_DIR=s3://ooni-public/sanitised
measurements/app.py:55:    if not config['REPORTS_DIR'].startswith("s3://"):
measurements/app.py:56:        raise Exception("REPORTS_DIR must start with s3://")
measurements/config.py:24:REPORTS_DIR = os.environ.get("REPORTS_DIR", "s3://ooni-public/sanitised/")
```

It seems to me we can get rid of this variable, no?

Reply:
Yeah that would be a good idea.